R/create.R

```diff
@@ -260,16 +260,16 @@ create_from_github <- function(repo_spec,
   invisible(proj_get())
 }
 
+# creates a backdoor we can exploit in tests
+allow_nested_project <- function() FALSE
```

Comment:
The new way we give ourselves of method of creating a nested project in tests. More self-explaining than previous reliance on `is_testing()` plus a magic string in the project name.